src/status_im/common/profile_picture_picker/view.cljs

```diff
@@ -86,5 +86,15 @@
                                :on-allowed  (fn [] (pick-pic update-profile-pic-callback))
                                :on-denied   (fn []
                                               (log/info
-                                               "user has denied permissions to select picture"))}))}]]])
+                                               "user has denied permissions to select picture"))}))}
+     (when has-picture?
+       {:accessibility-label :remove-profile-picture
+        :add-divider?        true
+        :danger?             true
+        :blur?               true
+        :icon                :main-icons/delete
```

Comment:
```suggestion
        :icon                :i/delete
```
We should use the `i` name instead of `main-icons` as it's the old one.
https://github.com/status-im/status-mobile/blob/0f43daa8368d0c7f4b48640eacde504562a877f6/doc/new-guidelines.md#icons